### ISSUE TITLE
[Q] Are images in `.ob-jupyter` ever automatically deleted?

### ISSUE THREAD

**Issue by NightMachinery** [emacs-jupyter/jupyter#502]
Are images in `.ob-jupyter` ever automatically deleted?

I had some images being lost, and the only culprit I can think of is auto-deletion to conserve storage.

**Comment by nnicandro** [comment]
Yes images are automatically deleted if there are some in the results of a code block and you execute the code block again. In that case, the new results would overwrite the old results so it made sense to delete the files of the old results.

See `org-babel-jupyter-cleanup-file-links`.
